go.mod

```diff
@@ -1,10 +1,14 @@
 module github.com/openfaas/mqtt-connector
 
-go 1.15
+go 1.18
```

Comment:
The go.mod version isn't important to update beyond 1.17 IMHO. The reason being is that setting it to 1.18 forces anyone referencing any of the code in this module to also use Go 1.18.